.github/scripts/snyk-report.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+
+KEYCLOAK_REPO="keycloak/keycloak"
+
+# Prevent duplicates by checking if a similar title exists
+check_github_issue_exists() {
+    local issue_title="$1"
+    # Extract the CVE ID
+    local CVE_ID=$(echo "$issue_title" | grep -oE '(CVE-[0-9]{4}-[0-9]{4,7}|SNYK-[A-Z]+-[A-Z0-9]+-[0-9]{4,7})')
+    local search_url="https://api.github.com/search/issues?q=$CVE_ID+is%3Aissue+sort%3Aupdated-desc+repo:$KEYCLOAK_REPO"
+    local response=$(curl -s -H "Authorization: token $GITHUB_TOKEN" -H "Accept: application/vnd.github.v3+json" "$search_url")
+    local count=$(echo "$response" | jq '.total_count')
+    
+    if [[ $count -gt 0 ]]; then
+        return 0  
+    else
+        return 1 
+    fi
+}
+
+# Create a GH issue based on the content of the CVE
+create_github_issue() {
+    local title="$1"
+    local body="$2"
+
+    local api_url="https://api.github.com/repos/$KEYCLOAK_REPO/issues"
+    local data=$(jq -n --arg title "$title" --arg body "$body" \
+                 '{title: $title, body: $body, labels: ["status/triage", "kind/cve", "status/important"]}')
```

Comment:
We don't have a label `status/important`, we have priority labels, but those should be set as part of triage, not automatically.

Reply:
@stianst updated